docs/common/DocsExample.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="docs-example">
+
+
+    <div class="code-toggle-button">
+      <KIconButton  class="code-toggle"
```

Comment:
Could you use `<KIconButton appearance='raised-button' ...>` Then you can remove `code-toggle` class. Note this will also change the button background color to light gray, which is desired. The yellow is a bit too much. Code toggle is not as important element on the documentation page overall. You can see [KIconButton docs](https://design-system.learningequality.org/kiconbutton/).